[2025-05-02T07:29:14.238+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-02T07:29:14.292+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_gcs.extract_customers manual__2025-05-02T07:28:54.729051+00:00 [queued]>
[2025-05-02T07:29:14.357+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_gcs.extract_customers manual__2025-05-02T07:28:54.729051+00:00 [queued]>
[2025-05-02T07:29:14.371+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-02T07:29:14.511+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-05-02 07:28:54.729051+00:00
[2025-05-02T07:29:14.555+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=320) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-02T07:29:14.564+0000] {standard_task_runner.py:63} INFO - Started process 322 to run task
[2025-05-02T07:29:14.567+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_gcs', 'extract_customers', 'manual__2025-05-02T07:28:54.729051+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/mysql_to_gcs_dag.py', '--cfg-path', '/tmp/tmp3x07royt']
[2025-05-02T07:29:14.571+0000] {standard_task_runner.py:91} INFO - Job 34: Subtask extract_customers
[2025-05-02T07:29:14.804+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_gcs.extract_customers manual__2025-05-02T07:28:54.729051+00:00 [running]> on host dfbad22d9104
[2025-05-02T07:29:15.441+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_gcs' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-05-02T07:28:54.729051+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-02T07:28:54.729051+00:00'
[2025-05-02T07:29:15.446+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-02T07:29:15.460+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-05-02T07:29:15.463+0000] {mysql_to_gcs_dag.py:19} INFO - Starting extraction for table: customers, timestamp: 20250502_072854
[2025-05-02T07:29:15.555+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2025-05-02T07:29:17.145+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:263: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2025-05-02T07:29:18.299+0000] {mysql_to_gcs_dag.py:32} INFO - Saved CSV to: /tmp/mysql_extract/customers_data_20250502_072854.csv
[2025-05-02T07:29:18.338+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-02T07:29:18.353+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-05-02T07:29:21.647+0000] {gcs.py:560} INFO - File /tmp/mysql_extract/customers_data_20250502_072854.csv uploaded to bronze_layer/customers_data_20250502_072854.csv in mysql-to-gcs-bronze-layer bucket
[2025-05-02T07:29:21.670+0000] {mysql_to_gcs_dag.py:41} INFO - Uploaded customers_data_20250502_072854.csv to GCS bucket.
[2025-05-02T07:29:21.683+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-02T07:29:21.685+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-02T07:29:22.087+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_gcs, task_id=extract_customers, run_id=manual__2025-05-02T07:28:54.729051+00:00, execution_date=20250502T072854, start_date=20250502T072914, end_date=20250502T072922
[2025-05-02T07:29:22.268+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-02T07:29:22.327+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
